projects/storefrontlib/layout/a11y/keyboard-focus/trap/trap-focus.service.ts

```diff
@@ -57,11 +57,11 @@ export class TrapFocusService extends TabFocusService {
     }
   }
 
-  protected getTrapStart(trap: TrapFocusType): boolean {
+  protected getTrapStart(trap?: TrapFocusType): boolean {
```

Comment:
I don't think we should make trap optional. Similar reason as [this](https://github.com/SAP/spartacus/pull/15658/files#r870749898)